packages/SwingSet/misc-tools/replay-transcript.js

```diff
@@ -515,116 +393,14 @@ async function replay(transcriptFile) {
-    if (vso[0] === 'vatstoreGet') {
-      const response = knownVCSyscalls.get(vso[1]);
-
-      if (!response) {
-        throw new Error(`Unknown vc vatstore entry ${vso[1]}`);
-      }
-
-      return response;
-    }
-
-    throw new Error(`Unexpected syscall ${vso[0]}(${vso.slice(1).join(', ')})`);
-  };
-
-  /**
-   * @param {WorkerData} workerData
-   * @returns {import('../src/kernel/vat-loader/transcript.js').CompareSyscalls}
-   */
-  const makeCompareSyscalls = workerData => {
-    const doCompare = (
-      _vatID,
-      originalSyscall,
-      newSyscall,
-      originalResponse,
-    ) => {
-      const error = bestRequireIdentical(vatID, originalSyscall, newSyscall);
-      if (
-        error &&
-        JSON.stringify(originalSyscall).indexOf('error:liveSlots') !== -1
-      ) {
-        return undefined; // Errors are serialized differently, sometimes
-      }
-
-      if (error) {
-        console.error(
-          `during transcript num= ${lastTranscriptNum} for worker PID ${workerData.xsnapPID} (start delivery ${workerData.firstTranscriptNum})`,
-        );
-
-        if (
-          // @ts-expect-error may be a symbol in some versions
-          error === extraSyscall &&
-          !argv.skipExtraVcSyscalls
-        ) {
-          return new Error('Extra syscall disallowed');
-        }
-      }
-
-      const newSyscallKind = newSyscall[0];
-
-      if (
-        // @ts-expect-error may be a symbol in some versions
-        error === missingSyscall &&
-        !argv.simulateVcSyscalls
-      ) {
-        return new Error('Missing syscall disallowed');
-      }
-
-      if (
-        argv.simulateVcSyscalls &&
-        supportsRelaxedSyscalls &&
-        !error &&
-        (newSyscallKind === 'vatstoreGet' ||
-          newSyscallKind === 'vatstoreSet') &&
-        vcSyscallRE.test(newSyscall[1])
-      ) {
-        if (newSyscallKind === 'vatstoreGet') {
-          if (originalResponse !== undefined) {
-            knownVCSyscalls.set(newSyscall[1], originalResponse);
-          } else if (!knownVCSyscalls.has(newSyscall[1])) {
-            console.warn(
-              `Cannot store vc syscall result for vatstoreGet(${newSyscall[1]})`,
-            );
-            knownVCSyscalls.set(newSyscall[1], undefined);
-          }
-        } else if (newSyscallKind === 'vatstoreSet') {
-          knownVCSyscalls.set(newSyscall[1], ['ok', newSyscall[2]]);
-        }
-      }
-
-      return error;
-    };
-    const compareSyscalls = (
-      _vatID,
-      originalSyscall,
-      newSyscall,
-      originalResponse,
-    ) => {
+  const wrapWithTimeTracker = (handler, workerData) => {
+    const wrapped = vso => {
       updateDeliveryTime(workerData);
-      const result = doCompare(
-        _vatID,
-        originalSyscall,
-        newSyscall,
-        originalResponse,
-      );
-      reportWorkerResult({
-        xsnapPID: workerData.xsnapPID,
-        result,
-        originalSyscall,
-        newSyscall,
-      });
```

Comment:
Interesting way of neutering the syscall results checks logic in this file.

Reply:
yeah, sorry, I was in a hurry :(